custom_components/tahoma/tahoma_device.py

```diff
@@ -109,14 +105,45 @@ def device_info(self) -> Dict[str, Any]:
         manufacturer = self.select_state(CORE_MANUFACTURER_NAME_STATE) or "Somfy"
         model = self.select_state(CORE_MODEL_STATE) or self.device.widget
 
-        return {
-            "identifiers": {(DOMAIN, self.unique_id)},
-            "manufacturer": manufacturer,
-            "name": self.name,
-            "model": model,
-            "sw_version": self.device.controllable_name,
+        name = self.name
+        device_url = self.device_url
```

Comment:
From what I see, you don't need this. You never use it. The next time we see it, you change its value [here]( https://github.com/iMicknl/ha-tahoma/pull/343/files#diff-14ad169bb8b77afd973fb894391dd4b0e629c3ebef801e129e618e09ce79a444R115) is enough.

Reply:
We need it for devices that have no subdevices. I will see if I can make the code more clear.

Reply:
At least perhaps rename it to something like `main_device_url`?